src/css/_sidebar.scss

```
@@ -0,0 +1,203 @@
+    }
+
+    &--active {
+      background: none;
+      color: var(--ifm-color-primary);
+      font-weight: 500;
+    }
+  }
+
+  // Infima draws these at 2rem, which reads heavy against 15px labels. Kept at
+  // full strength — at this size it's faint rather than loud.
+  //
```

Stylelint (17.14.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)